package.json

```diff
@@ -66,7 +66,7 @@
     "patternfly": "3.20.0",
     "pluralize": "5.0.0",
     "rxjs": "5.4.1",
-    "syndesis.data.mapper": "https://github.com/syndesisio/syndesis-datamapper-ui.git#9476d91ec03c0e4885f3005876c67216130ef98d",
+    "syndesis.data.mapper": "https://github.com/atlasmap/atlasmap-ui.git",
```

Comment:
ok i can do that, let me find the latest hash there and make sure it builds local.